README.md

```diff
@@ -26,7 +26,7 @@ npm install -g @nerdwallet/shepherd
 If using GitHub Enterprise, ensure the following environment variables are exported:
 
 ```
-export SHEPHERD_GITHUB_ENTERPRISE_BASE_URL={company_github_enterprise_base_url} # e.g., api.github.com
+export SHEPHERD_GITHUB_ENTERPRISE_BASE_URL={company_github_enterprise_base_url} # e.g., github.com
```

Comment:
Should we add both as example Github enterprise server (github.com) and api.github.com (Cloud github enterprise)? @jonathanmorley

Reply:
Im not sure api.github.com is ever appropriate here.
I am using GitHub Enterprise Cloud, and github.com is the right value for this setting.
I would expect GitHub Enterprise Server to be installation-dependent.

Reply:
When working with GitHub, the API endpoints differ based on whether you are using GitHub's public cloud service, GitHub Enterprise Cloud, or GitHub Enterprise Server (on-premises). Here’s how the endpoints vary:

* GitHub Public (GitHub.com)*: https://api.github.com/.
* GitHub Enterprise Cloud*: https://api.github.com/enterprise/[enterprise-name]
* GitHub Enterprise Server (On-Premises): https://[hostname]/api/v3/

In each case, the API functionality and how you interact with it are largely the same, but the base URL changes based on where your GitHub instance is hosted.

Reply:
I think what you are describing is the API endpoint for GitHub, which is configurable in shepherd with `SHEPHERD_GITHUB_ENTERPRISE_URL`.

However, this environment variable is `SHEPHERD_GITHUB_ENTERPRISE_BASE_URL`, which appears to only be used as a hostname to generate git clone URLs.
I have observed both public github.com and github enterprise cloud using `github.com` for these values.

Reply:
I've observed this as well using Enterprise Cloud